packages/editor/src/components/parts/web-service/Permission.test.tsx

```diff
@@ -1,16 +1,58 @@
-import { CollapsableUtil, SelectUtil, render } from 'test-utils';
+import { CollapsableUtil, SelectUtil, render, screen } from 'test-utils';
 import { Permission } from './Permission';
 
+const defaultConfig = { anonymous: true, error: 'Everybody', role: 'ivy:security:forbidden' };
+
 describe('Permission', () => {
   test('data', async () => {
-    render(<Permission />, { wrapperProps: { data: { config: { permission: { error: '>> Ignore Exception', role: 'Test' } } } } });
+    render(
+      <Permission
+        anonymousFieldActive={false}
+        config={{ anonymous: true, error: 'bla error', role: 'Test' }}
+        defaultConfig={defaultConfig}
+        updatePermission={() => {}}
+      />
+    );

```

Comment:
I would create a reusable method with some params for the different usage states